README.md

```diff
@@ -19,7 +19,7 @@ It also integrates with Windows Explorer and Microsoft Visual Studio (2010/2012/
   <tr>
     <td>Requirement</td>
     <td>MS Windows 7SP1+ <br/>MS .NET Framework 4.6.1+</td>
-    <td><a href="http://www.mono-project.com/download/">Mono 5.0+</a> <br /><small>(limited support due to Mono WinForms implementation)</s,all></td>
+    <td><a href="http://www.mono-project.com/download/">Mono 5.0+</a> <br /><small>(depreciated support due to Mono WinForms implementation)</s,all></td>
```

Comment:
* Typo (an existing) `small`
* Typo (new) `deprecated`. `Depreciate` is an accounting term meaning `to lose value`. I've made the same mistake myself few times, hehe
* I think it is worth directing users to v2.51 release for Linux-compatible version. The readme will need to be updated with a meaningful content to provide clarity around versions etc. We will do it separately.